metaflow/plugins/airflow/exceptions.py

```diff
@@ -0,0 +1,20 @@
+from metaflow.exception import MetaflowException
+
+
+class AirflowNotPresent(MetaflowException):
+    headline = "Airflow cannot be imported"
+
+    def __init__(self):
+        msg = "Airflow dependency is missing. Please install airflow to use the airflow cli command"
+        super().__init__(msg)
+
+
+class AirflowException(MetaflowException):
+    headline = "Airflow Exception"
+
+    def __init__(self, msg):
+        super().__init__(msg)
+
+
+class NotSupportedException(MetaflowException):
```

Comment:
This exception doesn't seem to be used right now anywhere.

Reply:
Used when we detect foreachs.